server/mustache-templates/landingPage.mst

```diff
@@ -34,34 +32,16 @@
                     {{#clients}}
                         <tr class="table-data table-secondary">
                             <th scope="row"><a href="/clients/{{id}}">{{displayName}}</a></th>
-                            <td></td>
-                            <td></td>
                             <td>{{#isSendingAlerts}}y{{/isSendingAlerts}}{{^isSendingAlerts}}NOPE{{/isSendingAlerts}}</td>
                             <td>{{#isSendingVitals}}y{{/isSendingVitals}}{{^isSendingVitals}}NOPE{{/isSendingVitals}}</td>
                         </tr>
-                        {{#locations}}
-                            <tr class="table-data">
-                                <td></td>
-                                <th scope="row"><a href="/locations/{{id}}">{{name}}</a></th>
-                                <td>{{sessionStart}}</td>
-                                <td>{{#isSendingAlerts}}y{{/isSendingAlerts}}{{^isSendingAlerts}}NOPE{{/isSendingAlerts}}</td>
-                                <td>{{#isSendingVitals}}y{{/isSendingVitals}}{{^isSendingVitals}}NOPE{{/isSendingVitals}}</td>
-                            </tr>
-                        {{/locations}}
                     {{/clients}}
                     </tbody>
                 </table>
             </div>
         </div>
     </div>
     
-    <script>
-        function reloadPage() {
-            location.reload(true)
```

Comment:
Hmm, I think this script should be kept - location isn't referring to a sensor I'm pretty sure.

This reloads the page every 30 seconds